content/docs/guides/partner-billing.md

```diff
@@ -0,0 +1,346 @@
+
+```bash
+curl --request GET \
+     --url https://console.neon.tech/api/v2/consumption/projects?limit=2 \
+     --header 'accept: application/json' \
+     --header 'authorization: Bearer $NEON_API_KEY' | jq'
+```
+And here is a sample response (with key lines highlighted):
+<details>
+<summary>Response body</summary>
+<CodeBlock highlight="16-17,31,47-48,62,66-67">
+```json
+{
+  "projects": [
+    {
+      "id": "wispy-wind-94231251",
+      "platform_id": "aws",
+      "region_id": "aws-us-east-2",
+      "name": "UserProjectRenamed",
+      "provisioner": "k8s-pod",
+      "default_endpoint_settings": {
+        "autoscaling_limit_min_cu": 1,
+        "autoscaling_limit_max_cu": 1,
+        "suspend_timeout_seconds": 0
+      },
+      "settings": {
+        "quota": {
+          "active_time_seconds": 200000000,
+          "compute_time_seconds": 18000
+        }
+      },
+      "pg_version": 15,
+      "proxy_host": "us-east-2.aws.neon.tech",
+      "branch_logical_size_limit": 204800,
+      "branch_logical_size_limit_bytes": 214748364800,
+      "store_passwords": true,
+      "active_time": 0,
+      "cpu_used_sec": 0,
+      "creation_source": "console",
+      "created_at": "2023-10-29T16:48:31Z",
+      "updated_at": "2023-11-01T11:44:56Z",
+      "synthetic_storage_size": 32605352,
+      "quota_reset_at": "2023-11-01T00:00:00Z",
+      "owner_id": "cf80d675-bb06-4e0c-9327-48ccfe84e7be"
+    },
+    {
+      "id": "divine-tree-77657175",
+      "platform_id": "aws",
+      "region_id": "aws-us-east-2",
+      "name": "RnameII",
+      "provisioner": "k8s-pod",
+      "default_endpoint_settings": {
+        "autoscaling_limit_min_cu": 1,
+        "autoscaling_limit_max_cu": 1,
+        "suspend_timeout_seconds": 0
+      },
+      "settings": {
+        "quota": {
+          "active_time_seconds": 36000,
+          "compute_time_seconds": 18000
+        }
+      },
+      "pg_version": 15,
+      "proxy_host": "us-east-2.aws.neon.tech",
+      "branch_logical_size_limit": 204800,
+      "branch_logical_size_limit_bytes": 214748364800,
+      "store_passwords": true,
+      "active_time": 0,
+      "cpu_used_sec": 0,
+      "creation_source": "console",
+      "created_at": "2023-10-29T16:33:39Z",
+      "updated_at": "2023-10-31T17:08:13Z",
+      "synthetic_storage_size": 32671528,
+      "quota_reset_at": "2023-11-01T00:00:00Z",
+      "owner_id": "cf80d675-bb06-4e0c-9327-48ccfe84e7be"
+    }
+  ],
+  "pagination": {
+    "cursor": "patient-frost-50125040"
+  }
+}
+```
+</CodeBlock>
+</details>
+
+
+## Resetting a project after suspend
+Generally, projects remain suspended until the next billing period. It is good practice to notify your users when they are close to reaching a limit; if the user is then suspended and loses access to their database, it will not be unexpected. If you have configured no further actions, the user will have to wait until the next billing period starts to resume usage.
```

Comment:
It says generally here. Are there exceptions?

Reply:
The exception -- I think -- is if the partner does something to reset. I'll remove "generally" though. It could be misleading.